traffic_portal/test/end_to_end/users/users-spec.js

```diff
@@ -17,10 +17,26 @@
  * under the License.
  */
 
+var pd = require('./pageData.js');
 var cfunc = require('../common/commonFunctions.js');
 
 describe('Traffic Portal Users Test Suite', function() {
+	const pageData = new pd();
 	const commonFunctions = new cfunc();
+	const myNewUser = {
+		username: 'user-' + commonFunctions.shuffle('abcdefghijklmonpqrstuvwxyz0123456789'),
+		fullName: 'test-' + commonFunctions.shuffle('abcdefghijklmonpqrstuvwxyz0123456789'),
+		email: 'test@cdn.viper.com',
+		roleName: 'admin',
+		tenantId: ' - root',
```

Comment:
this really doesn't look like a tenant id

Reply:
it is.. but i'll use the dropdown value.